docs/sql/array.md

```diff
@@ -43,6 +43,7 @@ as `CREATE TABLE data AS (CITIES VARCHAR ARRAY, COUNTRY VARCHAR)`.
 | `ARRAY_JOIN`                                                         | Another name for `ARRAY_TO_STRING`                                                                                                                                                                                            |                                                                     |
 | `ARRAY_LENGTH(` _array_ `)`                                          | Another name for `CARDINALITY`                                                                                                                                                                                                | `ARRAY_LENGTH( ARRAY [1, 2, 3])` => `3`                             |
 | `ARRAY_POSITION(` _array_ `, ` _element_ `)`                         | Returns the (1-based) index of the first _element_ of the _array_ as a long. Returns 0 if _element_ doesn't exist in _array_. If the _element_ is `NULL`, `NULL` is returned.                                                 | `ARRAY_POSITION(ARRAY [1, 3, 4, 6], 4)` => 3                        |
+| `ARRAY_REMOVE(` _array_ `, ` _element_ `)`                           | Removes all elements that are equal to _element_ from the array                                                                                                                                                               | `ARRAY_REMOVE(ARRAY [2, 2, 6, 8], 2)` => `[6, 8]`                   |
```

Comment:
wasn't there a funny behavior for nulls?

Reply:
For element as null, we return null. Same for CAST(null as integer array).

Reply:
yes, but this behavior has to be documented